$NetBSD: patch-af,v 1.1 2008/02/15 16:23:09 gdt Exp $

Fix a bug in processing the --with-ssh-security option.  Already fixed
in amanda svn, just backported from there.

--- configure.orig	2007-06-06 19:22:45.000000000 -0400
+++ configure
@@ -13538,7 +13538,10 @@ echo $ECHO_N "checking SSH options... $E
     # now convert that to a comma-separated list of C strings
     eval "set dummy ${SSH_OPTIONS}"; shift
     SSH_OPTIONS=''
-    for i in "${@}"; do SSH_OPTIONS="${SSH_OPTIONS}${SSH_OPTIONS:+, }\"${i/\"/\\\"}\""; done
+    for i in "${@}"; do
+	quoted="\"`echo "$i" | sed -e 's/\"/\\\"/'`\""
+	SSH_OPTIONS="${SSH_OPTIONS}${SSH_OPTIONS:+, }$quoted";
+     done
     echo "$as_me:$LINENO: result: $SSH_OPTIONS" >&5
 echo "${ECHO_T}$SSH_OPTIONS" >&6
 
@@ -62491,7 +62494,7 @@ fi;
 # Check whether --with-xsltproc-flags or --without-xsltproc-flags was given.
 if test "${with_xsltproc_flags+set}" = set; then
   withval="$with_xsltproc_flags"
-   if test "x$withval" == "xno"; then
+   if test "x$withval" = "xno"; then
 	XSLTPROC_FLAGS=''
     else
 	if test "x$withval" != "xyes"; then
